app/server/update-controller/src/main/java/io/syndesis/server/update/controller/bulletin/IntegrationUpdateHandler.java

```diff
@@ -319,7 +319,7 @@ private Connection includeConnector(Connection connection) {
                             dataManager.update(dbConnection);
                         }
 
-                        if (connection.getId().isPresent()) {
+                        if (connection.getId().isPresent() && ! integration.isDeleted()) {
```

Comment:
Put this change in another pull request.

Reply:
Sneaked in ... bit late to change now.